proteinworkshop/datasets/base.py

```diff
@@ -518,7 +530,7 @@ def get(self, idx: int) -> Data:
         :return: PyTorch Geometric Data object.
         """
         if self.in_memory:
-            return self._batch_format(self.data[idx])
+            return self._batch_format(copy.deepcopy(self.data[idx]))
```

Comment:
Does this lead to one of the performance enhancements we discussed previously?

Reply:
No noticeable delta but it makes me feel better about some of the gotchas here: https://alecstashevsky.com/post/on-the-fly-augmentation-with-pytorch-geometric-and-lightning-what-tutorials-dont-teach/

Reply:
Nice. SGTM